arbor/include/arbor/mechcat.hpp

```diff
@@ -70,9 +72,7 @@ class mechanism_catalogue {
     // Construct a schema for a mechanism derived from an existing entry,
     // with a sequence of overrides for global scalar parameter settings
     // and a set of ion renamings.
-    void derive(const std::string& name, const std::string& parent,
-                const std::vector<std::pair<std::string, double>>& global_params,
-                const std::vector<std::pair<std::string, std::string>>& ion_remap = {});
+    void derive(const std::string& name, const std::string& parent, const std::vector<std::pair<std::string, double>>& global_params, const std::vector<std::pair<std::string, std::string>>& ion_remap = {});
```

Comment:
Format issue unrelated to this PR: this looks a bit too long to me. Maybe we should reconsider the clang-format rules?

Reply:
That's actually horrible, yes. I tweaked the style to 128 columns and `AlignAfterBraces = true`.